src/stylus/product-collection/hero-header.styl

```diff
@@ -7,7 +12,7 @@
 		height 400px
 		
 .product_collection-header-category
-	font-size 3.5em
+	font-size calc(minfont + (strip(maxfont) - strip(minfont)) * (100vw - minscreen) / (strip(maxscreen) - strip(minscreen)))
```

Comment:
`strip()` is a POSTCSS plugin, you just need to add the  font-size without the unit, `remove-unit` might work with stylus?